src/mongo/client/dbclient.cpp

```diff
@@ -2110,16 +2127,18 @@ void DBClientBase::update(
         0, "update document exceeds maxBsonObjectSize", obj.objsize() <= getMaxBsonObjectSize());
     updates.enqueue(new UpdateWriteOperation(query.obj, obj, flags));
 
+    bool bypassDocumentValidation = flags & UpdateOption_BypassDocumentValidation;
+
     WriteResult writeResult;
-    _write(ns, updates.ops, true, wc, &writeResult);
+    _write(ns, updates.ops, true, bypassDocumentValidation, wc, &writeResult);
 }
 
 BulkOperationBuilder DBClientBase::initializeOrderedBulkOp(const std::string& ns) {
-    return BulkOperationBuilder(this, ns, true);
+    return BulkOperationBuilder(this, ns, true, false);
```

Comment:
I guess these were just helper functions for end users? OK then. I suppose we could also add a defaulted parameter for bypass to them? Not sure it is worth it. We can always do it later if people need it. I doubt they will.